### PR TITLE
updating draft text

### PR DESCRIPTION
**Pull request by aguoietf**

### CODE REVIEW

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 1]
```diff
@@ -320,66 +464,90 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
...
 
+   A network slice customer has the capability to configure customer
+   intent topologies without needing any prior knowledge of the
+   provider's network or resource availability.  However, this approach
+   could potentially create challenges for the provider in understanding
+   and realizing the intended topology.
+
+   Alternatively, the provider can choose to describe the available
+   resources and capabilities in the form of an abstract topology, which
+   is then exposed to the customer before network slice requests.  By
```
Comment (sergiobelotti) [inline draft-liu-teas-transport-network-slice-yang.txt:532]: We need to be clear here what this abstract topology should be: I mean, before we said that "a minimum level of information for the provider is needed for the customer to make a request for an intent topology. Now we are talking about abstract topo exposed by provider. This surely makes sense but we need to clarify which level this abstract topology is. Is the same level of information we have talked about before in the I-D? E.G. SAP topology ?

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 2]
```diff
@@ -320,66 +464,90 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
...
+   negotiations between the customer and the provider.  The process and
+   the data models for the provider to expose abstract topologies are
+   outside the scope of this document.
+
+   The provider communicates the operational state of the topology,
+   reflecting the allocated resources that result from negotiations
+   between the customer and the provider.  Subsequently, customers can
+   process the requested topology and seamlessly integrate it into their
+   own network topology.  Importantly, this relationship between the
+   customer and provider can be recursive.  For instance, a customer who
```
Comment (sergiobelotti) [inline draft-liu-teas-transport-network-slice-yang.txt:545]: I'm a bit confused here form the process: which network topology is the "own network topology"? Let me try to rephrase a bit ;
" subsequently , customer can process the network topology get form operational datastore (with resource allocation proposal from provider) and integrate that on the customer intent topology ." 
Is correct my proposal?

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 3]
```diff
@@ -485,215 +627,238 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
...
+             +--:(link-hop)
+             |  +--rw link-id?   nt:link-id
+             +--:(tp-hop)
+                +--rw tp-id?     nt:tp-id
+     augment /ietf-nss:network-slice-services/ietf-nss:slice-service
+               /ietf-nss:connection-groups/ietf-nss:connection-group
+               /ietf-nss:connectivity-construct/ietf-nss:slo-sle-policy
+               /ietf-nss:custom/ietf-nss:service-slo-sle-policy
+               /ietf-nss:sle-policy/ietf-nss:steering-constraints:
+       +--rw disjointness?   te-types:te-path-disjointness
```
Comment (sergiobelotti) [inline draft-liu-teas-transport-network-slice-yang.txt:809]: We maintain that even if still in debate , right ?

File: ietf-ns-topo.yang [hunk 4]
```diff
@@ -278,9 +315,9 @@
              "/ietf-nss:steering-constraints" +
              "/ietf-nss:path-constraints" {
        description
-         "Add toplogy id and explicit path to the connection group";
+         "Underlay path for connection group.";
```
Comment (sergiobelotti) [inline ietf-ns-topo.yang:318]: for connection group or connectivity-construct?

File: ietf-ns-topo.yang [hunk 5]
```diff
@@ -312,10 +349,9 @@
              "/ietf-nss:steering-constraints" +
              "/ietf-nss:path-constraints" {
        description
-         "Add toplogy id and explicit path to the connectivity
-		  construct";
+         "Underlay path for connection group.";
```
Comment (sergiobelotti) [inline ietf-ns-topo.yang:352]: connection-group ? not connectivity-construct ..?

**Review by sergiobelotti — APPROVED** [review]

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 6]
```diff
@@ -86,112 +91,124 @@ Table of Contents
...
+   intent topologies.
+
+   In the context of network slicing, customer intent topologies enables
+   customers to express resource reservation preferences.  These
+   topologies allow flexible configuration and activation of network
+   slices on demand.  By providing full control over resource allocation
+   timing and methods, customer intent topologies ensure that resources
+   are consistently available.  Moreover, the resources reserved via
+   customer intent topologies can be shared across network slices
+   created at different times or between different connections within
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply draft-liu-teas-transport-network-slice-yang.txt:140]: yes it should be connectivity-constructs per latest NS NBI

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 7]
```diff
@@ -86,112 +91,124 @@ Table of Contents
...
+   other representing the hub site - connected via a shared link, we
+   proactively reserve resources for the shared connection.  This
+   ensures that bandwidth is readily available whenever the customer
+   requires it.  In contrast, achieving equivalent bandwidth assurance
+   through individual dedicated connections would necessitate creating
+   separate links between each spoke and the hub, which would lead to
+   substantial bandwidth inefficiency.
+
+   Customer intent topology complements connectivity-based network
+   slicing by providing customers a mechanism to to specify additional
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply draft-liu-teas-transport-network-slice-yang.txt:159]: fixed.

File: draft-liu-teas-transport-network-slice-yang.txt (hunk 1 above)
Comment (aguoietf) [reply draft-liu-teas-transport-network-slice-yang.txt:532]: I think the minimum level of information referred before is required for network slicing, while the abstract topology referred here is totally optional and up to the provider.

File: draft-liu-teas-transport-network-slice-yang.txt (hunk 2 above)
Comment (aguoietf) [reply draft-liu-teas-transport-network-slice-yang.txt:545]: Yes the topology here refers to the customer intent topology, not the abstract topology which is outside the scope of the document.

File: ietf-ns-topo.yang (hunk 5 above)
Comment (aguoietf) [reply ietf-ns-topo.yang:352]: This augment is for connection group, which applies to all connectivity-constructs with the group. 

There is a separate augment for connectivity constructs, which applies to a single connectivity construct.

The two has different scope.

File: draft-liu-teas-transport-network-slice-yang.txt [hunk 8]
```diff
@@ -214,23 +238,16 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
...
+      (specifically, the Network Slice Controller or NSC).  It
+      represents the customer's desired network topology.
 
    *  Abstract Topology: A topology exposed to the customer by the
       network slice service provider prior to the creation of network
       slices.  The provider uses an abstract topology to expose useful
       information, such as available resources to the customer, which
-      can facilitate the build-up of customized topologies by the
+      can facilitate the build-up of customer intent topologies by the
       customer.
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply draft-liu-teas-transport-network-slice-yang.txt:251]: Sure. I added a word "optionally" in the description:
The provider may **optionally** uses an abstract topology to expose useful information, such as available resources to the customer, which can facilitate the build-up of customer intent topologies by the customer.

**Pull request merged**